python/dgl/distributed/dist_graph.py

```diff
@@ -345,14 +345,6 @@ def __init__(self, server_id, ip_config, num_servers,
             # Create the graph formats specified the users.
             self.client_g = self.client_g.formats(graph_format)
             self.client_g.create_formats_()
-            # Sort underlying matrix beforehand to avoid runtime overhead during sampling.
```

Comment:
BTW, removing this code may cause some breaking changes to users who are using the old partitioned graph. They will get errors like the graph is not sorted. I guess this is not avoidable but we need to let @zheng-da know about the changes.

Reply:
yes. you're right.